enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -15,15 +19,22 @@ import (
 	"github.com/buildbuddy-io/buildbuddy/server/interfaces"
 	"github.com/buildbuddy-io/buildbuddy/server/util/alert"
 	"github.com/buildbuddy-io/buildbuddy/server/util/background"
+	"github.com/buildbuddy-io/buildbuddy/server/util/grpc_client"
 	"github.com/buildbuddy-io/buildbuddy/server/util/log"
+	"github.com/buildbuddy-io/buildbuddy/server/util/lru"
 	"github.com/buildbuddy-io/buildbuddy/server/util/networking"
+	"github.com/buildbuddy-io/buildbuddy/server/util/perms"
 	"github.com/buildbuddy-io/buildbuddy/server/util/random"
 	"github.com/buildbuddy-io/buildbuddy/server/util/status"
 
+	regpb "github.com/buildbuddy-io/buildbuddy/proto/registry"
 	repb "github.com/buildbuddy-io/buildbuddy/proto/remote_execution"
 )
 
 var (
+	imageStreamingRegistryGRPCTarget = flag.String("executor.podman_image_streaming.registry_grpc_target", "", "gRPC endpoint of BuildBuddy registry")
```

Comment:
wdyt about `s/podman_/podman./` in the flag name? (I added a flag `executor.podman.enable_stats` in separate PR, so that podman namespace exists now)